### ISSUE TITLE
would please mention in the readme.txt that the size of the input image should be dividable by 16

### ISSUE THREAD

**Issue by MoHassoubah** [Beckschen/TransUNet#35]
Hello,

Many thanks for your contribution and sharing you work

would please mention in the readme.txt that the side of the input image should be dividable by 16, so others can know if they want to use image size other than 224x224?

thanks,

**Comment by andife** [comment]
interesting. What happens if the images do not have the appropriate divisors?

**Comment by MoHassoubah** [comment]
> interesting. What happens if the images do not have the appropriate divisors?

well, you get an error 
in file vit_seg_modeling.py, class Embeddings, forward function  in this line "embeddings = x + self.position_embeddings" about a mismatching size
and also an error here
in file vit_seg_modeling.py , class DecoderBlock, forward function, line "x = torch.cat([x, skip], dim=1)" also error of mismatching dimensions

**Comment by MoHassoubah** [comment]
> interesting. What happens if the images do not have the appropriate divisors?

well, you get an error 
in file vit_seg_modeling.py, class Embeddings, forward function  in this line "embeddings = x + self.position_embeddings" about a mismatching size
and also an error here
in file vit_seg_modeling.py , class DecoderBlock, forward function, line "x = torch.cat([x, skip], dim=1)" also error of mismatching dimensions

**Comment by andife** [comment]
Interesting, I had a different image dimension, but still I saw no error
